lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -2600,8 +2627,14 @@ GLStatus TFGraphLowering::lowerConditionalRegion(ConditionalSESERegion *r) {
 
   // Remember each of the results so that any references to the SIL BBArguments
   // that got defined end up referring to this node.
-  for (int i = 0, e = trueCodeFn.outputs.size(); i != e; ++i)
-    addValueMapping({trueCodeFn.outputs[i].first, 0}, {result, i});
+  graphFn.tentative_outputs.clear();
+  for (int i = 0, e = trueCodeFn.outputs.size(); i != e; ++i) {
+    TF_Output outputNode = {result, i};
+    // TODO: should we use 0 or i below?
+    addValueMapping({trueCodeFn.outputs[i].first, 0}, outputNode);
+    graphFn.tentative_outputs.push_back(
```

Comment:
Not sure if we will need to do something similar for while regions. In any case, this should be good temporary fix for https://bugs.swift.org/browse/SR-8164